community-operators/jaeger/jaeger.v1.8.2.clusterserviceversion.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: jaeger-operator.v1.8.2
   namespace: placeholder
   annotations:
-    categories: "tracing, monitoring, troubleshooting"
+    categories: "Logging & Tracing"
```

Comment:
It appears that the change to the CSV is only reflected in this repository and not in the [jaeger-operator repository](https://github.com/jaegertracing/jaeger-operator/pulls) and I assume this is the same for all the CSV changes.

This change seems to be a result from an OpenShift driven effort to minimize the number of category options displayed in the UI. I don't believe that we've defined a process for introducing changes originating from OpenShift directives - one possible solution could be requesting changes on future PRs submitted by the authors of the Operator or we could require @robszumski to go and update each of the CSVs in the operator repos. @aravindhp, could you please let me know what you think and how you would like us to proceed.

Reply:
It's not OpenShift driven, it's operatorhub,io driven